qmkshrobj does not cause an error in the configure script with GCC 4.2.
Later, however, it does cause an error in the build phase with GFortran.
--- configure~	2018-11-21 10:38:39.000000000 -0700
+++ configure	2019-01-27 12:16:37.000000000 -0700
@@ -68002,7 +68002,7 @@
 
 cat confdefs.h - <<_ACEOF >conftest.$ac_ext
 /* end confdefs.h.  */
-
+#error "-qmkshrobj should not work"
 	#include <stdio.h>
                           const char hw[] = "Hello, World\n";
 int
